index.html

```diff
@@ -1,33 +1,62 @@
 <!doctype html>
 
 <html lang="en">
-<head>
-  <meta charset="utf-8">
 
-  <title>Sprint Challenge - UI / RWD</title>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sprint Challenge - UI / RWD</title>
 
-  <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
-  <link rel="stylesheet" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
+    <link rel="stylesheet" href="css/index.css">
 
 </head>
 
 <body>
     <div class="container">
+        <header>
+            <nav>
+                <ul>
+
+                    <a href="http://#">Home</a>
+
+
+                    <a href="http://#">About</a>
+
+
+                    <a href="http://#">Products</a>
+
+
+                    <a href="http://#">Blog</a>
+
+
+                    <a href="http://#">Contact</a>
+
+                </ul>
+            </nav>
+            <img src="img/lambda-black.png" alt="">
+        </header>
+
         <section class="top-content">
+            <img src="/img/jumbo.jpg" alt="Overexposure shot of cars">
```

Comment:
You may be calling the image incorrectly. I do not see it when I open this in the browser.